internal/resources/azure/util.go

```diff
@@ -45,3 +45,12 @@ func contains(a []string, x string) bool {
 func regexPtr(regex string) *string {
 	return strPtr(fmt.Sprintf("/%s/i", regex))
 }
+
+func doesStrSliceContains(a []string, x string) bool {
```

Comment:
As above, we use `contains` elsewhere for this function so might want to consider being consistent in case we refactor it.

Reply:
Same as above. I'll apply it whenever when we come up with a final decision